addons/pos_restaurant/static/tests/tours/pos_restaurant.js

```diff
@@ -165,14 +171,130 @@ registry.category("web_tour.tours").add("pos_restaurant_sync_second_login", {
+        ProductScreen.clickOrderButton(),
+        ProductScreen.orderlinesHaveNoChange()
+    ].flat(),
+});
+
+function mergeTableHelpers() {
+    return [
+        FloorScreen.clickTable("5"),
+        ProductScreen.controlButton("Merge"),
+        {
+            content: `click the merge button`,
+            trigger: 'i[aria-label="Merge"]',
+        },
+        FloorScreen.clickTable("4"),
+        FloorScreen.backToFloor(),
+        {
+            content: `Verify table 4 and 5 is merge`,
+            trigger: 'div.table div.label:contains("4")',
+            isCheck: true,
+            run: () => {
+                if ($("div.table div.label:contains('4')").length < 2) {
+                    throw new TourError("Table isn't merge");
+                }
+            }
+        },
+    ]
+}
+
+function checkMergeTableIsCancelHelpers() {
+    return [
+        {
+            content: `Verify table 4 and 5 isn't merge anymore`,
+            trigger: 'div.table div.label:contains("4")',
+            isCheck: true,
+            run: () => {
+                if ($("div.table div.label:contains('4')").length !== 1) {
+                    throw new TourError("Table is still merge");
+                }
+            }
+        },
+        {
+            content: `Verify table 4 and 5 isn't merge anymore`,
+            trigger: 'div.table div.label:contains("5")',
+            isCheck: true,
+            run: () => {
+                if ($("div.table div.label:contains('5')").length !== 1) {
+                    throw new TourError("Table is still merge");
+                }
+            }
+        },
+    ]
+}
+
+registry.category("web_tour.tours").add("MergeTableTour", {
+    test: true,
+    url: "/pos/ui",
+    steps: () => [
+        Dialog.confirm("Open session"),
+        ...mergeTableHelpers(),
+        FloorScreen.clickTable("4"),
+        ProductScreen.clickDisplayedProduct("Coca-Cola"),
+        ProductScreen.clickPayButton(),
+        PaymentScreen.clickPaymentMethod("Cash"),
+        PaymentScreen.clickValidate(),
+        {
+            ...Dialog.confirm(),
+            content:
+                "acknowledge printing error ( because we don't have printer in the test. )",
+        },
+        ReceiptScreen.clickNextOrder(),
+        ...checkMergeTableIsCancelHelpers(),
+        ...mergeTableHelpers(),
+        {
+            content: `click on the burger menu`,
+            trigger: 'i.fa-bars',
+        },
+        {
+            content: `click on Edit Plan in the burger menu`,
+            trigger: 'a.dropdown-item:contains("Edit Plan")',
+        },
+        {
+            content: `select linked table`,
+            trigger: 'div.isLinked div.label:contains("4")',
+        },
+        {
+            content: `unlink in edit plan if unlink possible`,
+            trigger: '.edit-buttons button:contains("Unlink")',
+        },
+        {
+            content: `click on the burger menu`,
+            trigger: 'i.fa-bars',
+        },
```

Comment:
you can use the `clickMenuButton` function from `point_of_sale/static/tests/tours/helpers/ChromeTourMethods.js`